search/utils.py

```diff
@@ -176,10 +183,21 @@ def search_index(tokens=None, repo_slug=None, sort_by=None, terms=None):
         # Always sort by ID to preserve ordering.
         search = search.sort(sort_by, "id")
 
-    terms = set(get_vocab_names(repo_slug=repo_slug))
-    terms.update(set(('run', 'course', 'resource_type')))
+    terms = set(get_vocab_slugs(repo_slug=repo_slug))
```

Comment:
The variables `term` and `terms` in this line and the line below should be changed to `vocab` and `vocabs`.